app/src/main/java/fr/free/nrw/commons/profile/leaderboard/UserDetailAdapter.java

```diff
@@ -1,23 +1,32 @@
 package fr.free.nrw.commons.profile.leaderboard;
 
+import android.accounts.Account;
+import android.accounts.AccountManager;
 import android.content.Context;
 import android.net.Uri;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 import android.widget.TextView;
+import android.widget.Toast;
 import androidx.annotation.NonNull;
 import androidx.recyclerview.widget.RecyclerView;
+import butterknife.internal.DebouncingOnClickListener;
 import com.facebook.drawee.view.SimpleDraweeView;
+import fr.free.nrw.commons.BuildConfig;
 import fr.free.nrw.commons.R;
 
+
 /**
  * This class extends RecyclerView.Adapter and creates the UserDetail section of the leaderboard
  */
 public class UserDetailAdapter extends RecyclerView.Adapter<UserDetailAdapter.DataViewHolder> {
 
     private LeaderboardResponse leaderboardResponse;
 
+    //stores the username of current login
+    private String  currentLoginUserName="";
```

Comment:
Rather than an empty String, how about `null`? And modify `if (currentLoginUserName.equals` to avoid null pointer exceptions.